protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -396,7 +396,7 @@ message MergeRepositoryCommitsRequest {
     //for a or b only one of commit or branch should be populated
     string branch_a = 5;
     string branch_b =6;
-
+    bool isDryRun = 7;
```

Comment:
Unrelated. Let's not block one change because of discussions in another change.

Also, follow the same case style of the other fields.

Reply:
Ok, I'll extract it in the other PR.